notebooks/Feature Overview.ipynb

```diff
@@ -571,13 +601,20 @@
     "\"\"\").compute() \n",
     "t.set_index([\"target\", \"species\"]).unstack(\"species\").number.plot.bar()"
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
   }
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "dask-sql",
```

Comment:
I think it is probably more general to keep the original "Python" kernel name, as not every user might have named their kernel "dask-sql" (and it will also work better with Binder).

Reply:
sure, makes sense,  I will revert this changes back to previous config